products/jse/features/jse_metatag.feature

```diff
@@ -0,0 +1,78 @@
+Feature: Ensure Site Owners can create, edit and delete metatags. Ensure editors can create, edit and delete metatags
+  In order to ensure that site owners can create, edit and delete metatags
+  As a site owner
+  I want to be able create, edit and delete metatags
+  As an editor
+  I want to be able to edit metatags
+
+  @api @dev @destructive @javascript
+  Scenario: Ensure Site owners can create, edit and delete their own metatags
```

Comment:
We don't need to test for "own metatags." This is an administration page that affects the whole site.